README.md

```diff
@@ -27,6 +27,10 @@ app.on('error', function (err) {
   console.log(err.message);
 });
 
+app.on('processing_error', function (err) {
```

Comment:
Just one more request...

Can we leave this as `error`? It's in the example because if you don't listen to `error` events on an EventEmitter it'll throw the exception and crash your app. I'm ok with leaving the new event out of the main example.